cmake/FindQtRuntime.cmake

```diff
@@ -0,0 +1,44 @@
+# Find a compatible Qt version
+# Inputs: WITH_SERVER, WITH_CLIENT, WITH_ORACLE, WITH_DBCONVERTER, FORCE_USE_QT5
+# Outputs: COCKATRICE_QT_VERSION_NAME -- Example values: Qt5, Qt6
+
+set(COCKATRICE_QT_COMPONENTS Core Core5Compat)
+if(WITH_SERVER)
+    set(COCKATRICE_QT_COMPONENTS ${COCKATRICE_QT_COMPONENTS} Network Sql WebSockets)
+endif()
+if(WITH_CLIENT)
+    set(COCKATRICE_QT_COMPONENTS ${COCKATRICE_QT_COMPONENTS} Concurrent Multimedia Network PrintSupport Svg Widgets WebSockets)
+endif()
+if(WITH_ORACLE)
+    set(COCKATRICE_QT_COMPONENTS ${COCKATRICE_QT_COMPONENTS} Concurrent Network Svg Widgets)
+endif()
+if(WITH_DBCONVERTER)
+    set(COCKATRICE_QT_COMPONENTS ${COCKATRICE_QT_COMPONENTS} Network Widgets)
+endif()
+list(REMOVE_DUPLICATES COCKATRICE_QT_COMPONENTS)
+
+if(NOT FORCE_USE_QT5)
+    find_package(Qt6 6.2.3 COMPONENTS ${COCKATRICE_QT_COMPONENTS} QUIET HINTS ${Qt6_DIR})
+endif()
+if(Qt6_FOUND)
+    set(COCKATRICE_QT_VERSION_NAME Qt6)
+else()
+    list(REMOVE_ITEM COCKATRICE_QT_COMPONENTS Core5Compat) # Core5Compat is Qt6 Only
```

Comment:
why remove the item here if you could be adding it instead, resulting in less complexity?